src/lib/core.d.ts

```diff
@@ -1150,7 +1150,7 @@ interface ArrayConstructor {
     (arrayLength?: number): any[];
     <T>(arrayLength: number): T[];
     <T>(...items: T[]): T[];
-    isArray(arg: any): boolean;
+    isArray(arg: any): arg is Array<any>;

```

Comment:
We could, but we should add a separate overload for it. Otherwise, not passing the type argument gives you `Array<{}>`